trade-finance-manager-api/src/v1/controllers/utilisation-reports/get-fee-record-details-by-id.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import { HttpStatusCode, isAxiosError } from 'axios';
+import { FeeRecordDetailsResponseBody } from '../../api-response-types';
+import api from '../../api';
+
+type GetFeeRecordDetailsByIdResponse = Response<FeeRecordDetailsResponseBody | string>;
+
+export const getFeeRecordDetailsById = async (req: Request, res: GetFeeRecordDetailsByIdResponse) => {
```

Comment:
Please can we remove the 'byId'? I think by Id is generally the most standard way to fetch something and the majority of similar controllers don't seem to specify